client/src/assets/data/filterOptions.js

```diff
@@ -21,11 +21,11 @@ export default {
     className: "filter-3",
     options: [
       "Medical Supplies",
-      "Groceries/Food",
+      "Groceries / Food",
```

Comment:
Or maybe you did merge master but update the change made in 53a3583a987773eabb2216854cec8f0200d75558 ?

Anyways, let's keep no spaces in both f/e & model (no mapping).

Reply:
Sounds good to me, as long as there is consistency. I'll change it to no spaces. Thanks Josh!